src/pages/groupComparison/CopyNumberEnrichments.tsx

```diff
@@ -44,18 +44,23 @@ export default class CopyNumberEnrichments extends React.Component<ICopyNumberEn
     readonly enrichmentsUI = MakeMobxView({
         await:()=>[
             this.props.store.copyNumberData,
-            this.props.store.copyNumberEnrichmentProfile,
-            this.enrichmentAnalysisGroups
+            this.props.store.selectedStudyCopyNumberEnrichmentProfileSet,
+            this.enrichmentAnalysisGroups,
+            this.props.store.studies
         ],
         render:()=>{
             return (
                 <div data-test="GroupComparisonCopyNumberEnrichments">
-                    <EnrichmentsDataSetDropdown dataSets={this.props.store.copyNumberEnrichmentProfiles} onChange={this.onChangeProfile}
-                                                selectedValue={this.props.store.copyNumberEnrichmentProfile.result!.molecularProfileId}/>
+                    <EnrichmentsDataSetDropdown
+                        dataSets={this.props.store.copyNumberEnrichmentProfiles}
+                        onChange={this.onChangeProfile}
+                        selectedProfileByStudyId={this.props.store.selectedStudyCopyNumberEnrichmentProfileSet.result!}
+                        studies={this.props.store.studies.result!}
+                    />
                     <AlterationEnrichmentContainer data={this.props.store.copyNumberData.result!}
                         groups={this.enrichmentAnalysisGroups.result}
                         alteredVsUnalteredMode={false}
-                        headerName={this.props.store.copyNumberEnrichmentProfile.result!.name}
+                        headerName={"Copy number"}
```

Comment:
should the header depend on whether there is more than one profile? maybe for one profile, the header would be the profile name?